### PR TITLE
Add gasPrice, gasToken and ensName to FutureWallet

### PR DESCRIPTION
**Pull request by JustynaBroniszewska**

### CODE REVIEW

File: universal-login-sdk/src/core/services/WalletSerializer.ts [hunk 1]
```diff
@@ -19,6 +19,9 @@ export class WalletSerializer {
           wallet: {
             contractAddress: state.wallet.contractAddress,
             privateKey: state.wallet.privateKey,
+            ensName: state.wallet.ensName,
```
Comment (xitronix) [inline universal-login-sdk/src/core/services/WalletSerializer.ts]: Deduplicate ensName
```ts
{
kind: 'Future',
*name: state.name,
wallet: {
  *ensName**: state.wallet.ensName,
  ....
}
```

Comment (JustynaBroniszewska) [reply universal-login-sdk/src/core/services/WalletSerializer.ts]: Next PR?

File: universal-login-react/src/ui/commons/GasPrice/GasPrice.tsx [hunk 2]
```diff
@@ -62,10 +62,9 @@ export const GasPrice = ({isDeployed = true, deployedWallet, sdk, gasLimit, onGa
   };
 
   useEffect(() => {
-    const fastGasModeIndex = 1;
     if (gasModes) {
-      const {name, usdAmount} = gasModes[fastGasModeIndex];
-      const gasOption = findGasOption(gasModes[fastGasModeIndex].gasOptions, ETHER_NATIVE_TOKEN.address);
+      const {name, usdAmount} = gasModes[FAST_GAS_MODE_INDEX];
```
Comment (xitronix) [inline universal-login-react/src/ui/commons/GasPrice/GasPrice.tsx]: I think is better than introduce FAST_GAS_MODE_INDEX. It add for us flexibility. (we also can change gasModesName to enums)
const gasMode= findGasMode(gasModes, 'fast')

Comment (marekkirejczyk) [reply universal-login-react/src/ui/commons/GasPrice/GasPrice.tsx]: Why not just `defaultGasMode(gasModes)`?

**Pull request merged**